discordrb.json

```diff
@@ -57,6 +57,18 @@
             "voiceReceive": "no",
             "voiceSend": "yes",
             "voiceStateUpdate": "yes"
+        },
+        "misc": {
+            "supportsUserLogin": "yes",
+            "supportsBotLogin": "yes",
+            "cachesTokens": "yes",
+            "apiDocs": "inline",
+            "tutorials": "yes",
```

Comment:
+1 to urls. We shouldn't have to state that tutorials or examples exist or not... is there any lib that doesnt at least have examples? What is the distinction from tutorials and docs + examples? Etc.